scripts/config/jest/index.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
```

Comment:
So... move `scripts/jest` into `src/jest/cli`, then just require that from `scripts/jest`? I don't see much of a point, but works for me.